db/migrate/20170706230335_add_send_date_to_messages.rb

```diff
@@ -0,0 +1,5 @@
+class AddSendDateToMessages < ActiveRecord::Migration[5.0]
+  def change
+    add_column :messages, :send_date, :datetime
```

Comment:
Can we name it `:send_at` instead of `:send_date`, to conform with the other datetime columns in the table? (`:created_at`, `:updated_at`)